jetty-ee10/jetty-ee10-webapp/src/main/config/etc/jetty-ee10-webapp.xml

```diff
@@ -3,7 +3,6 @@
 
 <Configure id="Server" class="org.eclipse.jetty.server.Server">
   <Call class="org.eclipse.jetty.ee10.webapp.WebAppContext" name="addSystemClasses">
-    <Arg><Ref refid="Server"/></Arg>
```

Comment:
Why is this deleted?  This changes the configuration for all jetty servers in the JVM, rather than just the instance that this XML is configuring.   I agree it is unlikely that we have multiple servers when using jetty-home, but it is not impossible.